tests/all/integration/api/helpers/test_order.py

```diff
@@ -54,6 +56,117 @@ def test_should_delete_related_attendees(self):
             order = db.session.query(Order).filter(Order.id == obj.id).first()
             self.assertEqual(len(order.ticket_holders), 0)
 
+    def test_count_sold_tickets(self):
+        """Method to test the count query of sold tickets"""
+
+        with app.test_request_context():
+            # import logging
+            #
+            # logger = logging.getLogger('sqlalchemy.engine')
+            # logger.setLevel(logging.INFO)
+            ticket = TicketFactory()
+
+            first_completed_order = OrderFactory()
+            first_completed_order.status = 'completed'
+            second_completed_order = OrderFactory()
+            second_completed_order.status = 'completed'
+            third_completed_order = OrderFactory()
+            third_completed_order.status = 'completed'
+            fourth_completed_order = OrderFactory()
+            fourth_completed_order.status = 'completed'
+            fifth_completed_order = OrderFactory()
+            fifth_completed_order.status = 'completed'
+            sixth_completed_order = OrderFactory()
+            sixth_completed_order.status = 'completed'
+
+            placed_order = OrderFactory()
+            placed_order.status = 'placed'
+
+            first_initialized_order = OrderFactory()
+            first_initialized_order.created_at = datetime.utcnow()
+            second_initialized_order = OrderFactory()
+            second_initialized_order.created_at = datetime.utcnow() - timedelta(minutes=1)
+            third_initialized_order = OrderFactory()
+            third_initialized_order.created_at = datetime.utcnow() - timedelta(minutes=2)
+            fourth_initialized_order = OrderFactory()
+            fourth_initialized_order.created_at = datetime.utcnow() - timedelta(minutes=3)
+
+            first_expired_time_order = OrderFactory()
+            first_expired_time_order.created_at = common.date_
+            second_expired_time_order = OrderFactory()
+            second_expired_time_order.created_at = common.date_
+            third_expired_time_order = OrderFactory()
+            third_expired_time_order.created_at = common.date_
+
+            first_expired_order = OrderFactory()
+            first_expired_order.status = 'expired'
+            second_expired_order = OrderFactory()
+            second_expired_order.status = 'expired'
+            third_expired_order = OrderFactory()
+            third_expired_order.status = 'expired'
+            fourth_expired_order = OrderFactory()
+            fourth_expired_order.status = 'expired'
+            fifth_expired_order = OrderFactory()
+            fifth_expired_order.status = 'expired'
+
+            db.session.commit()
+
+            first_att_with_no_order_id = AttendeeFactoryBase()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'first_att_with_no_order_id' is assigned to but never used](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4726180)